reference/5.1/Microsoft.PowerShell.Core/About/about_pipelines.md

```diff
@@ -67,59 +67,46 @@ This pipeline is comprised of four commands in the specified order. The command
 is written horizontally, but we will show the process vertically in the following
 graphic.
 
-`Get-ChildItem` `-Path` *.txt
-
- **|**
-
-|   (FileInfo objects )
-|   (    .txt         )
-
- **|**
-
- **V**
-
-`Where-Object` {$_.length `-gt` 10000}
-
- **|**
-
-|   (FileInfo objects )
-|   (    .txt         )
-|   ( Length > 10000  )
-
- **|**
-
- **V**
-
+```powershell
+Get-ChildItem -Path '*.txt'
 
-`Sort-Object` `-Property` Length
+|
+|   ( FileInfo objects )
+|   (    .txt          )
+|
+V
 
```

Comment:
hm, the above should probably be in a plain codeblock (no powershell language indicator) so the ASCII art doesn't get syntax highlighted? but then you would need to alternate between PS code blocks and plain code blocks? or just have the entire thing in plain codeblocks...

Reply:
Another solution I thought of in hindsight, having a code block in one example, and another below that code block with the object types of the pipeline (instead of the commands)